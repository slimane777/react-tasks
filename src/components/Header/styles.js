import styled from 'styled-components';

export const Wrapper = styled.div`
    display: flex;
    background-color: #fbfbff;
    padding: 20px;
    box-shadow: 1px 0px 1px 0px grey;
`;