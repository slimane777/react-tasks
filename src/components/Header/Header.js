import React from 'react'
import { Wrapper } from './styles'
import ClearButton from '../Button/ClearButton';
import HeaderDate from '../Date/HeaderDate';

const Header = () => {
  return (
    <Wrapper>
      <HeaderDate />
      <ClearButton />
    </Wrapper>
  )
}

export default Header;