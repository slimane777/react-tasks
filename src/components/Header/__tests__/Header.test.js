import React from 'react';
import renderer from 'react-test-renderer';
import Header from '../Header';
import { Provider } from 'react-redux'
import configureStore from 'redux-mock-store'

describe('Header test', () => {
  const initialState = {tasks:{tasks: []}}
  const mockStore = configureStore()
  let store,wrapper

  it('Header should match snapshot', () => {
    store = mockStore(initialState)
    const component = renderer.create(<Provider store={store}><Header
      /></Provider>);
    const tree = component.toJSON();
    expect(tree).toMatchSnapshot();
  });
});
