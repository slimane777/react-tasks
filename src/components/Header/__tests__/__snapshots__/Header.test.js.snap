// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Header test Header should match snapshot 1`] = `
<div
  className="sc-bdnxRM bMDMPZ"
>
  <div
    className="sc-iCoGMd gtGxzP"
  >
    <div
      className="sc-jSFjdj eieNf"
    >
      <p
        className="sc-eCApnc iFHYB"
      >
        Tuesday,
      </p>
      <p
        className="sc-eCApnc bghzsz"
      >
         13th
      </p>
    </div>
    <p
      className="sc-gKAaRy jdnLDq"
    >
      July
    </p>
  </div>
  <div
    className="sc-hKFxyN erUByQ"
  >
    <span
      className="sc-dlnjwi gLYpaE"
    >
      0
       Tasks
    </span>
    <p
      className="sc-gtsrHT kFyifo clear"
      onClick={[Function]}
    >
      Clear List
    </p>
  </div>
</div>
`;
